.evergreen/install-node.sh

```diff
@@ -33,9 +33,18 @@ else
   [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
 
   set +x # nvm is very verbose
-  echo nvm install --no-progress $NODE_JS_VERSION && nvm alias default $NODE_JS_VERSION
-  nvm install --no-progress $NODE_JS_VERSION
-  nvm alias default $NODE_JS_VERSION
+
+  # A few distros where pre-built node20 does not work out of the box and hence
+  # needs to be built from source
+  if [[ "${DISTRO_ID}" =~ ^(amazon(1|2)-|debian9|rhel7|ubuntu18) ]] && [[ "$NODE_JS_VERSION" =~ ^20 ]];
+  then
+    . "$BASEDIR/install-node-source.sh"
```

Comment:
```suggestion
    bash "$BASEDIR/install-node-source.sh"
```

? Then you also don’t need to worry about variables from the script carrying over into the parent script here

Reply:
I see. Yea this makes sense. I will remove the cleanup in the script.